osu.Game.Rulesets.Osu/Difficulty/Preprocessing/OsuDifficultyHitObject.cs

```diff
@@ -68,20 +76,27 @@ private void setDistances()
                 scalingFactor *= 1 + smallCircleBonus;
             }
 
-            Vector2 lastCursorPosition = lastObject.StackedPosition;
-
-            var lastSlider = lastObject as Slider;
-            if (lastSlider != null)
-            {
-                computeSliderCursorPosition(lastSlider);
-                lastCursorPosition = lastSlider.LazyEndPosition ?? lastCursorPosition;
-
+            if (lastObject is Slider lastSlider)
                 TravelDistance = lastSlider.LazyTravelDistance * scalingFactor;
```

Comment:
This is now broken due to the removal of `computeSliderCursorPosition(lastSlider);`

Reply:
Has this been resolved?